packages/neuron-ui/src/components/MultisigAddress/index.tsx

```diff
@@ -204,6 +204,27 @@ const MultisigAddress = () => {
     [listActionOptions]
   )
 
+  const daoDisabledMessage = useMemo(() => {
+    if (!wallet.device) return ''
+
+    const multisigConfig = daoDepositAction.depositFromMultisig || daoWithdrawAction.withdrawFromMultisig
+    if (!multisigConfig) return ''
```

Comment:
`daoDepositAction.depositMultisig || daoWithdrawAction.withdrawFromMultisig is falsy` is covered by 

```ts
if (
      (daoDepositAction.depositFromMultisig && daoDepositAction.isDialogOpen) ||
      (daoWithdrawAction.withdrawFromMultisig && daoWithdrawAction.isDialogOpen)
   ) {
     return ...
   }
return ''
```